front/src/App.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import {
+  waitFor,
+  render,
+  screen,
+} from '@testing-library/react';
+import { HashRouter as Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { act } from 'react-dom/test-utils';
+import * as utils from './utils/Utils';
+import App from './App';
+import testUtils from './_TestUtils/TestUtils';
+
+const { date, week, schedule } = testUtils;
+
+describe('testing App', () => {
+  it('should render App', async () => {
+    utils.getSchedulingWeek = jest.fn(() => week);
+    utils.getSchedulingDate = jest.fn(() => schedule);
+    utils.validateLogin = jest.fn(() => Promise.resolve(false));
+    localStorage.setItem('token', 'dummy_token');
```

Comment:
Should this be set to something else now that the token is no longer in local storage?